acceptance_tests/cypress/integration/lotse_flow.spec.js

```diff
@@ -299,7 +299,7 @@ context('Acceptance tests', () => {
                 cy.get('#person_a_town').type(taxReturnData.personA.town)
```

Comment:
Not about this line, but general: Why did you not rename the messages to reflect the new field names? On purpose or did you forget?